pickATrip_django_apps/admin.py

```diff
@@ -23,11 +23,20 @@ def __init__(self, general_model, admin_site):
         super(ListAdminMixin, self).__init__(general_model, admin_site)
 
 
+def is_related_to_social_signing(suspected_model):
+    return suspected_model._meta.model.__name__ == "Site" or \
+           suspected_model._meta.model.__name__ == "EmailAddress" or \
+           suspected_model._meta.model.__name__ == "SocialApp" or \
+           suspected_model._meta.model.__name__ == "SocialToken" or \
+           suspected_model._meta.model.__name__ == "SocialAccount"
+
+
 # Register all other models automatically - should stay last in file.
 models = apps.get_models()
 for model in models:
     admin_class = type('AdminClass', (ListAdminMixin, admin.ModelAdmin), {})
-    try:
-        admin.site.register(model, admin_class)
-    except admin.sites.AlreadyRegistered:
-        pass
+    if not is_related_to_social_signing(model):
```

Comment:
This is not clear to me, Does the app could run both? 
with the social auth and without?

Reply:
no, It's somehow registered automatically, so In order to avoid conflicts of registering twice I used this bad solution, but If it makes too much trouble we can give up on this PR, the other mentors have told me it's not a great idea not in these words but it was the general opinion

Reply:
Sorry, i was asking misleading question,
In case no social media is set in admin, will it work the same w/o this PR?

Reply:
@Yarboa sorry I'm still not entirely understand your question, what do you mean by 'social media is set in admin'? I wasn't setting anything since Django is registering the social media models related to OAuth2 of google, and these specific models are making trouble with the auto-model registration code since it's trying to register the models again. I don't think I can pass on those settings (that generates these social media models of Django), I followed a tutorial so I believe every setting is crucial, and Django is having a very specific way to do so, as I read from several sources.

Reply:
There is pip install of django-allauth
And in App/url.py

This is default user, no option to work with non allauth.urls?

path('accounts/', include('allauth.urls'))

Reply:
I think for now I will put this PR in the backlog and later I'll try to rethink this issue, maybe after the test PRs will be merged and I will add then tests for checking these things you've mentioned if we decide as a team we would like to promote it.